pkg/handler/repositories.go

```diff
@@ -150,6 +150,10 @@ func (rh *RepositoryHandler) createRepository(c echo.Context) error {
 	newRepository.OrgID = &orgID
 	newRepository.FillDefaults()
 
+	if err = CheckSnapshotForRepo(c, newRepository); err != nil {
+		return err
+	}
```

Comment:
Shouldn't this check just gate whether or not a snapshot task is queued? I should still be able to create a repository, right? Same for bulkcreate.

Reply:
the check was for whether the user/account can snapshot the repo (i.e. via the feature configuration).  Maybe the name should be clearer?   'AllowSnapshotFeature' ?

Reply:
Right, but it's blocking a repo from being created because it's returning an error (if I don't have a snapshot feature access) and then the handler function is exiting.

Reply:
Oh ignore me! I'm testing this wrong. I had snapshot=true.